datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -122,33 +122,93 @@ CREATE TABLE dcsa_im_v3_0.party_identifying_code (
     code_list_name varchar(100)
 );
 
+DROP TABLE IF EXISTS dcsa_im_v3_0.payment_term_type CASCADE;
+CREATE TABLE dcsa_im_v3_0.payment_term_type (
+    payment_term_code varchar(3) PRIMARY KEY,
+    payment_term_name varchar(100) NOT NULL,
+    payment_term_description varchar(250) NOT NULL
+);
+
+DROP TABLE IF EXISTS dcsa_im_v3_0.inco_terms CASCADE;
+CREATE TABLE dcsa_im_v3_0.inco_terms (
+    inco_terms_code varchar(3) PRIMARY KEY,
+    inco_terms_name varchar(100) NOT NULL,
+    inco_terms_description varchar(250) NOT NULL
+);
+
+DROP TABLE IF EXISTS dcsa_im_v3_0.transport_document_type CASCADE;
+CREATE TABLE dcsa_im_v3_0.transport_document_type (
+    transport_document_type_code varchar(3) PRIMARY KEY,
+    transport_document_type_name varchar(20) NULL,
+    transport_document_type_description varchar(500) NULL
+);
+
+DROP TABLE IF EXISTS dcsa_im_v3_0.cut_off_times CASCADE;
+CREATE TABLE dcsa_im_v3_0.cut_off_times (
+    cut_off_times_code varchar(3) PRIMARY KEY,
+    cut_off_times_name varchar(100) NULL,
+    cut_off_times_description varchar(250) NULL
+);
+
+DROP TABLE IF EXISTS dcsa_im_v3_0.vessel CASCADE;
+CREATE TABLE dcsa_im_v3_0.vessel (
+    id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
+    vessel_imo_number varchar(7) NULL,
+    vessel_name varchar(35) NULL,
+    vessel_flag char(2) NULL,
+    vessel_call_sign_number varchar(10) NULL,
+    vessel_operator_carrier_id uuid NULL REFERENCES dcsa_im_v3_0.carrier (id)
+);
+
+DROP TABLE IF EXISTS dcsa_im_v3_0.communication_channel_qualifier CASCADE;
+CREATE TABLE dcsa_im_v3_0.communication_channel_qualifier (
+    communication_channel_qualifier_code varchar(2) PRIMARY KEY,
+    communication_channel_qualifier_name varchar(100) NOT NULL,
+    communication_channel_qualifier_description varchar(250) NOT NULL
+);
+
 DROP TABLE IF EXISTS dcsa_im_v3_0.booking CASCADE;
 CREATE TABLE dcsa_im_v3_0.booking (
-    carrier_booking_reference varchar(35) PRIMARY KEY,
-    receipt_delivery_type_at_origin varchar(3) NOT NULL REFERENCES dcsa_im_v3_0.receipt_delivery_type(receipt_delivery_type),
-    receipt_delivery_type_at_destination varchar(3) NOT NULL REFERENCES dcsa_im_v3_0.receipt_delivery_type(receipt_delivery_type),
+    id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
+    carrier_booking_request_reference varchar(100) NOT NULL,
```

Comment:
@HenrikHL should this not be `carrier_booking_reference` instead (to be consistent with what we refer to in the API specs) and it should be `UNIQUE` right? Based on our discussion I assumed that `id` and `carrier_booking_request_reference` are the same, if so should we combine the two? or did I misunderstand?